org.deluge_torrent.deluge.yaml

```diff
@@ -0,0 +1,456 @@
+
+- name: numpy
+  buildsystem: simple
+  build-commands:
+  - python3 setup.py build
+  - python3 setup.py egg_info
+  - python3 setup.py install --prefix=/app --root=/
+  sources:
+  - type: archive
+    url: https://files.pythonhosted.org/packages/c5/63/a48648ebc57711348420670bb074998f79828291f68aebfff1642be212ec/numpy-1.19.4.zip
+    sha256: 141ec3a3300ab89c7f2b0775289954d193cc8edb621ea05f99db9cb181530512
+
+- name: twisted
+  buildsystem: simple
+  build-commands:
+  - python3 setup.py build
+  - python3 setup.py egg_info
+  - python3 setup.py install --prefix=/app --root=/
+  sources:
+  - type: archive
+    url: https://files.pythonhosted.org/packages/4a/b4/4973c7ccb5be2ec0abc779b7d5f9d5f24b17b0349e23240cfc9dc3bd83cc/Twisted-20.3.0.tar.bz2
+    sha256: d72c55b5d56e176563b91d11952d13b01af8725c623e498db5507b6614fc1e10
+
+- name: pillow
+  buildsystem: simple
+  build-commands:
+  - python3 setup.py build
+  - python3 setup.py egg_info
+  - python3 setup.py install --prefix=/app --root=/
+  sources:
+  - type: archive
+    url: https://files.pythonhosted.org/packages/2b/06/93bf1626ef36815010e971a5ce90f49919d84ab5d2fa310329f843a74bc1/Pillow-8.0.1.tar.gz
+    sha256: 11c5c6e9b02c9dac08af04f093eb5a2f84857df70a7d4a6a6ad461aca803fb9e
+
+- name: chardet
+  buildsystem: simple
+  build-commands:
+  - python3 setup.py build
+  - python3 setup.py egg_info
+  - python3 setup.py install --prefix=/app --root=/
+  sources:
+  - type: archive
+    url: https://files.pythonhosted.org/packages/ee/2d/9cdc2b527e127b4c9db64b86647d567985940ac3698eeabc7ffaccb4ea61/chardet-4.0.0.tar.gz
+    sha256: 0d6f53a15db4120f2b08c94f11e7d93d2c911ee118b6b30a04ec3ee8310179fa
+
+- name: setproctitle
+  buildsystem: simple
+  build-commands:
+  - python3 setup.py build
+  - python3 setup.py egg_info
+  - python3 setup.py install --prefix=/app --root=/
+  sources:
+  - type: archive
+    url: https://files.pythonhosted.org/packages/a7/e1/e76b903af017cc4a99f823b5e4235c1136dcca6902e6b7ee4d545958ba8c/setproctitle-1.2.1.tar.gz
+    sha256: 5f0eecb27815e31799a69eb6a06b4d375d38887d079d410565b0be82da65c950
+
+- name: pyxdg
+  buildsystem: simple
+  build-commands:
+  - python3 setup.py build
+  - python3 setup.py egg_info
+  - python3 setup.py install --prefix=/app --root=/
+  sources:
+  - type: archive
+    url: https://files.pythonhosted.org/packages/6f/2e/2251b5ae2f003d865beef79c8fcd517e907ed6a69f58c32403cec3eba9b2/pyxdg-0.27.tar.gz
+    sha256: 80bd93aae5ed82435f20462ea0208fb198d8eec262e831ee06ce9ddb6b91c5a5
+
+- name: MarkupSafe
+  buildsystem: simple
+  build-commands:
+  - python3 setup.py build
+  - python3 setup.py egg_info
+  - python3 setup.py install --prefix=/app --root=/
+  sources:
+  - type: archive
+    url: https://files.pythonhosted.org/packages/b9/2e/64db92e53b86efccfaea71321f597fa2e1b2bd3853d8ce658568f7a13094/MarkupSafe-1.1.1.tar.gz
+    sha256: 29872e92839765e546828bb7754a68c418d927cd064fd4708fab9fe9c8bb116b
+
+- name: mako
```

Comment:
You can generate most of the python dependencies using https://github.com/flatpak/flatpak-builder-tools/tree/master/pip